modules/claims_api/spec/shared_reporting_helper.rb

```diff
@@ -0,0 +1,75 @@
+                                         cid: '0oa9uf05lgXYk6ZXn297',
+                                         evss_response: evss_response_array.to_json))
+  end
+  let(:pending_claims) { FactoryBot.create(:auto_established_claim, cid: '0oa9uf05lgXYk6ZXn297') }
+  let(:poa_submissions) do
+    poa_submissions = []
+    poa_submissions.push(FactoryBot.create(:power_of_attorney,
+                                           cid: '0oa9uf05lgXYk6ZXn297'))
+    poa_submissions.push(FactoryBot.create(:power_of_attorney,
+                                           cid: '0oa9uf05lgXYk6ZXn297'))
+    poa_submissions.push(FactoryBot.create(:power_of_attorney,
+                                           cid: '0oa9uf05lgXYk6ZXn297'))
+  end
+  let(:errored_poa_submissions) do
+    errored_poa_submissions = []
+    errored_poa_submissions.push(FactoryBot.create(:power_of_attorney, :errored, cid: '0oa9uf05lgXYk6ZXn297'))
+    errored_poa_submissions.push(FactoryBot.create(
+                                   :power_of_attorney,
+                                   :errored,
+                                   vbms_error_message: 'File could not be retrieved from AWS',
+                                   cid: '0oa9uf05lgXYk6ZXn297'
+                                 ))
+    errored_poa_submissions.push(FactoryBot.create(:power_of_attorney_without_doc, cid: '0oa9uf05lgXYk6ZXn297'))
+  end
+  let(:evidence_waiver_submissions) do
+    evidence_waiver_submissions = []
+    evidence_waiver_submissions.push(FactoryBot.create(:claims_api_evidence_waiver_submission,
+                                                       cid: '0oa9uf05lgXYk6ZXn297'))
+    evidence_waiver_submissions.push(FactoryBot.create(:claims_api_evidence_waiver_submission,
+                                                       cid: '0oa9uf05lgXYk6ZXn297'))
+    evidence_waiver_submissions.push(FactoryBot.create(:claims_api_evidence_waiver_submission,
+                                                       cid: '0oa9uf05lgXYk6ZXn297'))
+  end
+  let(:errored_evidence_waiver_submissions) do
+    errored_evidence_waiver_submissions = []
+    errored_evidence_waiver_submissions.push(FactoryBot.create(:claims_api_evidence_waiver_submission, :errored,
+                                                               cid: '0oa9uf05lgXYk6ZXn297'))
+    errored_evidence_waiver_submissions.push(FactoryBot.create(
+                                               :claims_api_evidence_waiver_submission,
+                                               :errored,
+                                               vbms_error_message: 'File could not be retrieved from AWS',
+                                               cid: '0oa9uf05lgXYk6ZXn297'
+                                             ))
+    errored_evidence_waiver_submissions.push(FactoryBot.create(:claims_api_evidence_waiver_submission,
+                                                               cid: '0oa9uf05lgXYk6ZXn297'))
+  end
+end
```

Comment:
These values are just refactored from the nightly job test file to be shared for both, nothing new added.